src/test/java/org/broadinstitute/hellbender/engine/FeatureDataSourceUnitTest.java

```diff
@@ -71,6 +72,13 @@ public void testGetSequenceDictionary() {
         }
     }
 
+    @Test (expectedExceptions = GATKException.class)
+    // this test asserts that a helpful exception is thrown for blockZipped files lacking an index as they may not be fully supported
+    //TODO this is a temporary fix until https://github.com/broadinstitute/gatk/issues/4224 has been resolved
+    public void testUnindexedBZippedFile() {
+        new FeatureDataSource<>(new File(toolsTestDir + "IndexFeatureFile/4featuresHG38Header.unindexed.vcf.gz"));
```

Comment:
In addition to this unit test, add an integration test to `FeatureSupportIntegrationTest` as well.

Reply:
done